extensions/widgets/navbar/tests/properties.livecodescript

```diff
@@ -0,0 +1,68 @@
+script "NavBarWidgetProperties"
+/*
+Copyright (C) 2015 - 2016 LiveCode Ltd.
+
+This file is part of LiveCode.
+
+LiveCode is free software; you can redistribute it and/or modify it under
+the terms of the GNU General Public License v3 as published by the Free
+Software Foundation.
+
+LiveCode is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or
+FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+for more details.
+
+You should have received a copy of the GNU General Public License
+along with LiveCode.  If not see <http://www.gnu.org/licenses/>.  */
+
+on TestSetup
+   TestLoadExtension "com.livecode.library.widgetutils"
+   TestLoadExtension "com.livecode.library.iconSVG"
+   TestLoadExtension "com.livecode.widget.navBar"
```

Comment:
I suspect the issue here is the capital b in navBar - these get used to build the paths to the extensions which are case-sensitive on linux.